bin/ncu

```diff
@@ -23,6 +23,7 @@ program
     .option('--cwd <path>', 'Used as current working directory for `spawn` in npm listing')
     .option('--dep <dep>', 'check only a specific section(s) of dependencies: prod|dev|peer|optional|bundle (comma-delimited)')
     .option('-e, --error-level <n>', 'set the error-level. 1: exits with error code 0 if no errors occur. 2: exits with error code 0 if no packages need updating (useful for continuous integration). Default is 1.', cint.partialAt(parseInt, 1, 10), 1)
+    .option('--engines-node', 'upgrade to latest version which satisfies engines.node range')
```

Comment:
You can omit 'latest' here since we are implementing `--engines-node` to be version target agnostic.